From 8ad73cb1121bae27805d51993c46dd1fc3821220 Mon Sep 17 00:00:00 2001
From: Markus Volk <f_l_k@t-online.de>
Date: Thu, 10 Aug 2023 12:13:05 +0200
Subject: [PATCH] install.sh: dont rename themes using spaces

---
 install.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/install.sh b/install.sh
index 0f5d5b0c..f546e12c 100755
--- a/install.sh
+++ b/install.sh
@@ -131,7 +131,7 @@ install_theme() {
   install -m644 "${SRC_DIR}/src/index.theme"                                     "${THEME_DIR}"
 
   # Update the name in index.theme
-  sed -i "s/%NAME%/${THEME_NAME//-/ }/g"                                         "${THEME_DIR}/index.theme"
+  sed -i "s/%NAME%/${THEME_NAME}/g"                                         "${THEME_DIR}/index.theme"
 
   if [ -z "${brightprefix}" ]; then
     cp -r "${SRC_DIR}"/src/{16,22,24,32,symbolic}                                "${THEME_DIR}"
-- 
2.41.0

